src/Files.App/Data/Items/NavigationBarSuggestionItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Items
+{
+	public class NavigationBarSuggestionItem : ObservableObject
```

Comment:
observing is really needed? theres no two way bindings though.

Reply:
Yes, we may reuse existing NavigationBarSuggestionItem instances instead of replacing them with new NavigationBarSuggestionItem instances when the item changes for smooth display.

Reply:
ok